include/date/tz.h

```diff
@@ -1158,7 +1158,7 @@ struct tzdb
 #if !USE_OS_TZDB
     std::vector<detail::Rule>   rules;
 #endif
-#ifdef _WIN32
+#if defined(_WIN32) || defined(RTC) 
```

Comment:
We make use of the Windows TZ mapping code, so we need this field on all platforms